src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/alert_response.cc

```diff
@@ -59,6 +59,17 @@ AlertResponse::~AlertResponse() {}
 void AlertResponse::Run() {
   LOG4CXX_AUTO_TRACE(logger_);
 
+  mobile_apis::Result::eType result_code =
+      static_cast<mobile_apis::Result::eType>(
+          (*message_)[strings::msg_params][strings::result_code].asInt());
+  ApplicationSharedPtr application = application_manager_.application(
```

Comment:
@IGapchuk application() method expects `uint32_t`, so i think connection key extracted from smart object should be casted to this type

Reply:
@mked-luxoft Fixed in 86f78e521e510b3a41cafe03b1b9bfed2c7b2433.